### PR TITLE
Add tabview

### PR DESCRIPTION
**Pull request by alexharb**
Removes code that prevented preview due to "Item" existing in sxdatamodeld

Adds tabview to switch between match status and Profile

### CODE REVIEW

File: Shared/ContentView.swift [hunk 1]
```diff
@@ -10,60 +10,15 @@ import CoreData
...
-        .toolbar {
-            #if os(iOS)
-            EditButton()
-            #endif
-
-            Button(action: addItem) {
-                Label("Add Item", systemImage: "plus")
-            }
+        return TabView(selection: /*@START_MENU_TOKEN@*//*@PLACEHOLDER=Selection@*/.constant(1)/*@END_MENU_TOKEN@*/) {
+            Text("Tab Content 1").tabItem { Text("Match Status") }.tag(1)
```
Comment (cd24) [inline Shared/ContentView.swift]: [Nit]
Each modifier should be on its own line to improve readability

**Review by cd24 — COMMENTED** [review]
Looks pretty good! I have only one small comment.

File: Shared/ContentView.swift (hunk 1 above)
Comment (alexharb) [reply Shared/ContentView.swift]: like this?

```suggestion
            Text("Tab Content 1")
               .tabItem { Text("Match Status") }
               .tag(1)
```

Comment (cd24) [reply Shared/ContentView.swift]: Yes, exactly.

Comment (alexharb) [reply Shared/ContentView.swift]: Fixed

**Review by cd24 — APPROVED** [review]
LGTM!

**Review by Allazzanni — APPROVED** [review]

**Pull request merged**